lib/gears/shape.lua

```diff
@@ -749,60 +749,119 @@ end
 -- @param cr A cairo context
 -- @tparam number w The shape width
 -- @tparam number h The shape height
+-- @tparam number r The arc radius 
 -- @tparam number percent The progressbar percent
 -- @tparam boolean hide_left Do not draw the left side of the shape
 -- @staticfct gears.shape.radial_progress
-function module.radial_progress(cr, w, h, percent, hide_left)
+function module.radial_progress(cr, w, h, radius, percent, hide_left)
```

Comment:
This breaks API. Anyone who already specifies a percent gets their config broken.

I guess this new parameter either needs to be moved to the end of the parameter list or a new function needs to be added that gets a `radius` argument.

(I did not look closely at the PR, but the rest looks fine to me (in a "first impression" sense))